### DISCUSSION TITLE
Secure radio config in app

### DISCUSSION THREAD

**Discussion by Bangs69** [meshtastic/firmware#5450] [Ideas]
We are running a private mesh on a separate channel.
The purpose of our mesh is to provide tracking and communication for parents and their school children (mainly during the transition from primary school to high school when they start walking to school and start increasing their independence). 
For this mesh to work it needs to be setup properly and all devices need to be in the correct mode. Our public mesh is an unusable mess because of incorrectly setup modules, wrong roles and old unmaintained devices sending out rubbish packets and taking hops that they can't possibly retransmit to anyone useful. It's a perfect example of how a mesh can turn to chaos without coordination. 

Our mesh works, as intended, because it was setup correctly and I have personally configured every single device on it myself with the strict agreement that they not enter the configuration panel under any circumstances and not share any mesh configuration details with ANYONE.

It has gone pretty well so far, a few instances where the children have messed about with the configuration,  but nothing serious until now. Someone has shared the configuration with an unknown person who has now joined the mesh with an improperly setup module role, so far the impact has been minor but if this spreads it will be a nightmare all over again.

Tl;dr: please add the option to pin code protect the configuration panel.

**Comment by garthvh** [comment]
There is already a managed mode that does exactly what you want

**Reply by Bangs69** [reply]
In managed mode they can't connect via Bluetooth to communicate

**Reply by garthvh** [reply]
This is incorrect.

**Reply by Bangs69** [reply]
I am using heltec v3.2, T-beam 1.2 and heltec wireless tracker modules. We did test managed mode on one of the v3.2's and although the device was operating perfectly we were unable to connect via Bluetooth, WiFi wasn't enabled and no settings we changed in the Web client would actually save.
This was a few weeks ago, perhaps a bug in an old firmware? 
In which case my apologies and appreciation for your help! We will update and try again.
